PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -2263,7 +2257,7 @@ int FindReplaceDlg::processRange(ProcessOperation op, FindReplaceInfo & findRepl
 			}
 			
 		}	
-		++nbProcessed;
+		if (processed) ++nbProcessed;

```

Comment:
Next time, please separate the instruction from the condition (put the instruction in a new line).

Reply:
@donho, @Uhf7 

Is the rule _always_ to put the instruction in a new line?
I think sometimes (like this one) it is very concise and very readable to do it the way @Uhf7 did it.
But if the rule is to always do it, then I suggest it should also use curly braces as well, like this:
```
if (processed)
{
    ++nbProcessed;
}
```
What is the thinking about that?

Reply:
> Is the rule always to put the instruction in a new line?

It depends.
In this case, it's just easier for reading (only for me maybe?).
On the following case, I would say keep it in the same line:
```
	switch (_winVersion)
	{
		case WV_WIN32S: return TEXT("Windows 3.1");
		case WV_95: return TEXT("Windows 95");
		case WV_98: return TEXT("Windows 98");
		case WV_ME: return TEXT("Windows Millennium Edition");
		case WV_NT: return TEXT("Windows NT");
		case WV_W2K: return TEXT("Windows 2000");
		case WV_XP: return TEXT("Windows XP");
		case WV_S2003: return TEXT("Windows Server 2003");
		case WV_XPX64: return TEXT("Windows XP 64 bits");
		case WV_VISTA: return TEXT("Windows Vista");
		case WV_WIN7: return TEXT("Windows 7");
		case WV_WIN8: return TEXT("Windows 8");
		case WV_WIN81: return TEXT("Windows 8.1");
		case WV_WIN10: return TEXT("Windows 10");
		default: /*case WV_UNKNOWN:*/ return TEXT("Windows unknown version");
	}
```
//But if the rule is to always do it, then I suggest it should also use curly braces as well, like this:
```
  if (processed)
  {
    ++nbProcessed;
  }
```

I'm not against such style, but I would use the following one:
```
  if (processed)
    ++nbProcessed;
```
so it looks more compact.

Reply:
I'm under the impression that increasing line count is negative for PR acceptance. All recommendations, I read so far, say so. It's an optimization criteria for PRs. So my single line can be seen as cheating here, I admit this. I'll avoid this in the future. Thank you for the hint.

Reply:
On a separate line also makes it more obvious what is going on when single stepping with the debugger.

Reply:
It's much easier to debug, if it is in two separate lines, I sign this immediately.

Reply:
> I'm under the impression that increasing line count is negative for PR acceptance. All recommendations, I read so far, say so.

True. I avoid the tourists and bad coders.
But the readability of codes you added is important as well, so "bad" coding style also brings some negative points :)

Reply:
"tourists" LOL

Reply:
And, @sasumner, to answer your actual question, I've been involved in projects already, where opening a new code block (opening curly brace after if's) was compulsory. So I kind of understand this wish.

But it's readable enough for me without the curly brackets, as long as the indentation is correct, and as long it is only a simple short statement. A multi-parameter call spreading over several lines deserves curly brackets.

 And, since the Notepad++ code contains already many of those single statements after if's, I would simply continue this tradition.

Reply:
> I'm under the impression that increasing line count is negative for PR acceptance. All recommendations, I read so far, say so.

Your statement is totally true in *.h BTW

Reply:
@Uhf7 The real reason to like the curly braces even with a single statement, is that when that statement becomes 2 or more, it is easy to fall into the trap of believing the indentation. :-)

Reply:
Believing in indentation? That could happen if a Python programmer is at work ...